### PR TITLE
Suppress a distracting warning (fixes #94)

### LINKED ISSUE #94: DeprecationWarning: invalid escape sequence \c in test/plugins/pylint_test.py
https://travis-ci.com/cclauss/edx-lint/jobs/180003543#L247-L248
```
test/plugins/pylint_test.py::test_that_we_can_test_pylint
  <unknown>:12: DeprecationWarning: invalid escape sequence \c
```
https://github.com/edx/edx-lint/blob/master/test/plugins/pylint_test.py

### PR DESCRIPTION
**Pull request by nedbat**

### CODE REVIEW

**Comment by nedbat** [comment]
@cclauss Do you approve?

File: test/plugins/pylint_test.py [hunk 1]
```diff
@@ -94,7 +95,10 @@ def test_that_we_can_test_pylint():
         # TODO is checked by an IRawChecker. #=C
         """
     msg_ids = "unused-import,redefined-builtin,anomalous-backslash-in-string,fixme"
```
Comment (cclauss) [inline test/plugins/pylint_test.py]: Add __invalid-escape__ to __msg_ids__ ?

Comment (nedbat) [reply test/plugins/pylint_test.py]: `invalid-escape` isn't a pylint message.  `anomalous-backslash-in-string` is.  The problem you reported was that Python is warning about `\c` while pytest is also warning about it.  The pytest violation is being handed in the test. We needed to suppress the Python warning.

**Comment by nedbat** [comment]
@iloveagent57 @jmbowman @cpennington ?

**Pull request merged**